test/test.answers

```diff
@@ -201,177 +201,177 @@ b4dfdb3f7cef6571968fadd3e586a19226a546b3d470cafb8b7c556e1d2d3bfd  test/results/t
 2fad02905ddac3b919184e84692e16385589637d0571855ad787647712beaba4  test/results/test0001-aarch64-binary
 237923bf50bfa40d39015c487ad90638dc97a843aa1fbb0bc7beaea35ea40c6f  test/results/test0001-amd64-binary
 eb7e7424fd91708c5e74402f3f41813949a12b2d101bb5f42fc3fdaab79c2a50  test/results/test0001-armv7l-binary
-88b87aafd8f309c6f7f96c3c9b0f6a78a07d024f9cf24314c26a16c522b54ac3  test/results/test0001-knight-native-binary
-4b41bf8d379108b10c835154a6826bd37b49feeb4c2fef89ae002153839eb625  test/results/test0001-knight-posix-binary
```

Comment:
Those checksum changes are due to swapped operands (`ADD R0 R0 R1` instead of `ADD R0 R1 R0`)